### PR TITLE
Update Experiment docs for default deployment

### PR DESCRIPTION
**Pull request by tyiuhc**
# Amplitude Developer Docs PR

## Description

Update Experiment docs for default deployment

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [X] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [X] My documentation follows the style guidelines of this project.
- [X] I previewed my documentation on a local server using `mkdocs serve`.
- [X] Running `mkdocs serve` didn't generate any failures.
- [X] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Review by bgiori — APPROVED** [review]
Hold off on any merge until we release the feature

File: docs/experiment/general/data-model.md [hunk 1]
```diff
@@ -20,12 +20,13 @@ Experiment uses the same projects which are required for Amplitude Analytics. As
 
 ## Deployments
 
-In Amplitude Experiment, a deployment serves a group of flags or experiments for use in an application. Deployments have an associated randomly generated **deployment key** which Experiment uses to identify the deployment and authorize requests to the evaluation servers. 
+In Amplitude Experiment, a deployment serves a group of flags or experiments for use in an application. Each [project](#projects) has a deployment using the project API key as the deployment key, available by default. On creation, experiment deployments have an associated randomly generated **deployment key** which Experiment uses to identify the deployment and authorize requests to the evaluation servers.
 
 !!!info "Client vs Server Deployments"
-    Deployments are either client or server deployments. Only server deployments can use flag configs for [local evaluation](./evaluation/local-evaluation.md). Don't share or make flag configs public.
+    Deployments are either client or server deployments. Client-side deployments should be used to initialize client-side SDKs, and server-side deployments should be used to initialize server-side SDKs or authorize requests to the Evaluation API.
```
Comment (markzegarelli) [inline docs/experiment/general/data-model.md:26]:
```suggestion
    Deployments are either client or server deployments. Use client-side deployments should to initialize client-side SDKs, and server-side deployments to initialize server-side SDKs or authorize requests to the Evaluation API.
```

File: docs/experiment/general/data-model.md [hunk 2]
```diff
@@ -20,12 +20,13 @@ Experiment uses the same projects which are required for Amplitude Analytics. As
...
+
+Deployments live within Amplitude Analytics projects, and a project can have multiple deployments. Deployments are generally named after the platform (client-side) or service (server-side) which variants are served to (for example: `android`, `ios`, `web`). The default project API key deployment is useful for getting started, however using explicit deployments for each platform or service is the best practice for larger organizations or teams that may share the same Amplitude project across multiple platforms for the same application.
```
Comment (markzegarelli) [inline docs/experiment/general/data-model.md:28]:
```suggestion
Deployments belong to Amplitude Analytics projects, and a project can have multiple deployments. Amplitude recommends that you name deployments after the platform (client-side) or service (server-side) to which Experiment serves variants (for example: `android`, `ios`, `web`). The default project API key deployment is useful for getting started, but using explicit deployments for each platform or service is the best practice for larger organizations or teams that may share the same Amplitude project across multiple platforms for the same application.
```

File: docs/experiment/guides/getting-started/index.md [hunk 3]
```diff
@@ -6,7 +6,6 @@ template: guide-first.html
...
 
-1. [Create a deployment](./create-a-deployment.md). A deployment serves a group of flags or experiments for use in an application.
-2. [Create a flag](./create-a-flag.md), and add a deployment. Flags alone are used for standard feature flagging without user analysis. Experiments are used for feature experimentation on users. Learn more about [flags and experiments](../../general/data-model/#flags-and-experiments).
-3. [Fetch variants](./fetch-variants.md) to evaluate a user for a flag.
-4. [Track exposure](./track-exposure.md) to find out whether a user viewed the variable experience from your feature flag. Exposure tracking is optional for feature flags which don't require analysis, but it's essential when running experiment.
\ No newline at end of file
+1. [Create a flag](./create-a-flag.md), and add a [deployment](../../general/data-model.md#deployments). Flags alone are used for standard feature flagging without user analysis. Experiments are used for feature experimentation on users. Learn more about [flags and experiments](../../general/data-model/#flags-and-experiments).
```
Comment (markzegarelli) [inline docs/experiment/guides/getting-started/index.md:9]:
```suggestion
1. [Create a flag](./create-a-flag.md), and add a [deployment](../../general/data-model.md#deployments). Use flags by themselves for standard feature flagging without user analysis. Use experiments for feature experimentation on users. Learn more about [flags and experiments](../../general/data-model/#flags-and-experiments).
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/experiment/general/data-model.md (hunk 1 above)
Comment (bgiori) [inline docs/experiment/general/data-model.md:26]:
```suggestion
    Deployments are either client or server deployments. Use client-side deployments to initialize client-side SDKs, and server-side deployments to initialize server-side SDKs or authorize requests to the Evaluation API.
```

File: docs/experiment/guides/getting-started/index.md (hunk 3 above)
Comment (bgiori) [reply docs/experiment/guides/getting-started/index.md:9]:
```suggestion
1. [Create a flag](./create-a-flag.md), and add a [deployment](../../general/data-model.md#deployments). Use flags for standard feature flagging without user analysis. Use experiments for feature experimentation on users. Learn more about [flags and experiments](../../general/data-model/#flags-and-experiments).
```

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**